q2_bowtie2/plugin_setup.py

```diff
@@ -51,10 +52,16 @@
         ("aligned_reads", SampleData[SequencesWithQuality]),  # type: ignore
         ("unaligned_reads", SampleData[SequencesWithQuality]),  # type: ignore
         ("bowtie2_alignment", SampleData[AlignmentMap]),  # type: ignore
+        ("read_features", FeatureTable[Frequency]),  # type: ignore
     ],  # type: ignore
     input_descriptions={},
     parameter_descriptions={"threads": "number of alignment threads to launch"},
-    output_descriptions={},
+    output_descriptions={
+        "aligned_reads": "Fastq files with aligned reads.",
+        "unaligned_reads": "Fastq files with unaligned reads.",
+        "bowtie2_alignment": "The bowtie2 alignment file.",
+        "read_features": "Dataframe containing the read features output by bowtie2.",
```

Comment:
These output descriptions shouldn't mention the underlying data type of the artifacts

Reply:
I have changed this now.